01-cloudformation/README.md

```diff
@@ -159,19 +159,27 @@ name.
 #### Question: Why YAML
 
 _Why do we prefer the YAML format for CFN templates?_
+YAML use less punctuation and should be easier to write and to read. YAML inherently provides some features, such as commenting, that aren't available in JSON.
 
 #### Question: Protecting Resources
 
 _What else can you do to prevent resources in a stack from being deleted?_
+Set the DeletionPolicy attribute to prevent the deletion of an individual resource at the stack level.
```

Comment:
Correct. You can also use IAM user/role policies to restrict the ability for users to delete/update a stack and it's resources.